.travis.yml

```diff
@@ -1,5 +1,7 @@
 language: java
-jdk: openjdk10
+jdk:
+    - oraclejdk11
+    - openjdk11
```

Comment:
I wouldn't build with both. It'll make the CI builds take 2x as long, and it's probably better in the end to keep things simple and tell everybody they just need to install OpenJDK and forget about the Oracle JDK going forward.